bundle/language_en.properties

```diff
@@ -1329,7 +1329,6 @@ angal.patient.no.btn
 angal.patient.note                                                                                     = Note
 angal.patient.consensus.consensus.txt                                                                  = Consent to treatment
 angal.patient.consensus.consensus.mandatory.msg                                                        = "Consent to treatment" is mandatory.
-angal.patient.consensus.administrative.txt                                                             = Administrative consent
 angal.patient.consensus.service.txt                                                                    = Consent for additional services
 angal.patient.consensus.border                                                                         = Informed consent
```

Comment:
During last [Developers Call (n.18)](https://openhospital.atlassian.net/wiki/spaces/OHC/pages/2222522369/2023-05-16+Developers+Call+n.18) and further discussion with professionals, we decided to put it more clearly, in this way:

```
The patient:
[  ] has viewed and approved the treatment consent form
[  ] consents to the use of the data for additional services
```